scripts/objects/asteroidManager.js

```diff
@@ -71,6 +70,23 @@ Game.objects.AsteroidManager = function (managerSpec) {
       x: Random.nextGaussian(managerSpec.maxX / 2, managerSpec.maxX / 4),
       y: Random.nextGaussian(managerSpec.maxY / 2, managerSpec.maxY / 4),
     }
+
+    // increment the asteroidsGenerated by 1 + any asteroids that will come about when 
+    // these asteroids are broken apart 
+    asteroidsMade += 1; 
+    if(sizeCategory > 1) {
+      asteroidsMade += 3 + (3 % sizeCategory);
+      if(sizeCategory > 2) { asteroidsMade += 3 * 4; } 
```

Comment:
This code is rather gross. At this point, it would be easier to put it into a switch statement and have that be good enough.